docs/resources/group_iam_binding.md

```diff
@@ -0,0 +1,51 @@
+---
+page_title: "Resource hcp_group_iam_binding - terraform-provider-hcp"
+subcategory: "Cloud Platform"
+description: |-
+  Updates the group's IAM policy to bind a role to a new member. Existing bindings are preserved.
+---
+
+# hcp_group_iam_binding (Resource)
+
+Updates the group's IAM policy to bind a role to a new member. Existing bindings are preserved.
+
+~> **Note:** `hcp_project_iam_policy` can not be used in conjunction with
+`hcp_project_iam_binding`.
+
+## Example Usage
+
+```terraform
+# Fetch a user from HCP
+data "hcp_user_principal" "example" {
+  email = "user@example.com"
+}
+
+# Lookup an existing group in HCP
+data "hcp_group" "example" {
+  resource_name = "group-name"
```

Comment:
I guess my thinking with this is that _both_ the full resource name or the resource name suffix could be used in this part. Curious of your thoughts on this! Happy to change it to just `group_name` if we need to 👍🏽

Reply:
> I guess my thinking with this is that both the full resource name or the resource name suffix could be used in this part.

I am worried this can cause confusion. Moreover, the term in other places [1] means the full name. I think this should be changed to `name` or `group_name`. Between the two my preference is `name` because the field is local to the resource but happy with other also.

[1] https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/data-sources/organization#resource_name, https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/data-sources/project#resource_name, https://registry.terraform.io/providers/hashicorp/hcp/latest/docs/resources/vault_secrets_app#resource_name

Reply:
That makes sense to me! I'll. go ahead and change it to `name` to be consistent 👍🏽